backend/app/views/spree/admin/promotions/calculators/weighted_rate/_fields.html.erb

```diff
@@ -0,0 +1,54 @@
+<%= fields_for "#{prefix}[calculator_attributes]", calculator do |f| %>
+  <%= f.label :preferred_amount %>
+  <%= render "spree/admin/shared/number_with_currency", f: f, amount_attr: :preferred_amount, currency_attr: :preferred_currency %>
+<% end %>
+
+<div class="field">
+  <p>
+    <%= admin_hint(
+      calculator.model_name.human,
+      """
+        <p>
+          This amount will distributed to line items weighted relative to their
```

Comment:
I'm going to go ahead and merge this but would be happy to see a PR to improve it from either you or @graygilmore